react-native-sdk/README.md

```diff
@@ -3,10 +3,48 @@
 
 ## Installation
  Inside your project, run `npm i @jitsi/react-native-sdk`.<br/><br/>Additionally if not already installed, the following dependencies need to be added:
- <br/>`npm i @react-native-async-storage/async-storage react-native-webrtc`
+<br/>`npm i @react-native-async-storage/async-storage react-native-webrtc`<br/><br/>
```

Comment:
The list is much longer so let's not specify, let's just say that the script will do it.